src/PerfView/StackViewer/StackWindow.xaml.cs

```diff
@@ -411,6 +412,8 @@ public void SetStackSource(StackSource newSource, Action onComplete = null)
                             cumMax / 1000000, controller.GetStartTimeForBucket((HistogramCharacterIndex)cumMaxIdx));
                     }
 
+                    RedrawFlameGraph();
```

Comment:
I would like this to be lazy.   Simply have a bit that marks that the flame graph is invalid at this point.  Then have an event when the canvas becomes visible that actually calls the RedrawFlamGraph.

This avoids a bunch of work that may never be used (if users never look at the flamegraph.